components/Layout/layout.module.scss

```diff
@@ -9,9 +9,7 @@
   border-style: none;
   font-family: $softwireFont;
 }
-body {
-  margin: 0;
-}
```

Comment:
is this deliberate?

(Not against it - don't like having tag based selectors in a css module - just a little surprised to see it here)

Reply:
I don't think it was doing anything, and it wasn't happy compiling so I just removed it